mindmeld/models/query_features.py

```diff
@@ -670,15 +670,15 @@ def _extractor(query, resources):
 
 
 @register_query_feature(feature_name="sys-candidates")
-def extract_sys_candidates(entities=None, **args):
+def extract_sys_candidates(**args):
     """
     Return an extractor for features based on a heuristic guess of numeric \
         candidates in the current query.
 
     Returns:
             (function) The feature extractor.
      """
-    del args
+    entities = [args['entities']] if 'entities' in args else None
```

Comment:
I assume this is to fix another bug in `extract_sys_candidates`? if so in the interest of expediency, should we split this off into its own PR?

Reply:
It isn't, it's a modification to the existing extractor. The change additionally enables passing a specific system entity to obtain only corresponding features. The default was to return features for all system entities in the query. This change allows for validating presence of that particular system entity.

Reply:
Reverted back to original, made changes in the way slot filling was calling this logic to support original behavior of this function.